src/test/java/BaseTest.java

```diff
@@ -1,12 +1,109 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+Homework17
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+main
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/222062520-cc094f03-a94b-4c59-a08a-42b001da2d73.png)

![image](https://user-images.githubusercontent.com/104761482/222062610-6f5324a3-ea36-4058-949c-b302da4286cf.png)

Reply:
I checked the screenshot on my end and doesn't show anything of this. No idea what should I do?

Reply:
I did, is it working now?
Curious what does this do by checkout ?

Reply:
> I did, is it working now? Curious what does this do by checkout ?

You need to fix bugs, commit and push

Reply:
How is it now?

Reply:
Great! 

![image](https://user-images.githubusercontent.com/104761482/222197313-88656db2-ff63-4b4f-a470-3b32d597485e.png)